tests/e2e/tests_runner.sh

```diff
@@ -93,6 +93,21 @@ run_non_tee_tests() {
 
 }
 
+run_kata_qemu_se_tests() {
+	local aa_kbc="${1:-"offline_fs_kbc"}"
+	export AA_KBC="${aa_kbc}"
+
+	local runtime_config_file="/opt/confidential-containers/share/defaults/kata-containers/"
```

Comment:
I wonder if there is an opportunity to refactor this code with the `run_non_tee_tests` code, which is almost identical?